python/tidy.py

```diff
@@ -38,6 +38,7 @@
     os.path.join(".", "target", "*"),
     os.path.join(".", "ports", "gonk", "src", "native_window_glue.cpp"),
     os.path.join(".", "ports", "cef", "*"),
+    os.path.join(".", "ports", "geckolib", "*"),
```

Comment:
r=me with this line remove. I’ve tested locally that `./mach test-tidy` reports no error on this PR’s branch without it.